packages/rainbowkit/src/wallets/walletConnectors/ledgerWallet/ledgerWallet.ts

```diff
@@ -12,36 +38,163 @@ export interface LedgerWalletOptions {
 export const ledgerWallet = ({
   chains,
   projectId,
-}: LedgerWalletOptions): Wallet => ({
-  id: 'ledger',
-  iconBackground: '#000',
-  name: 'Ledger Live',
-  iconUrl: async () => (await import('./ledgerWallet.svg')).default,
-  downloadUrls: {
-    android: 'https://play.google.com/store/apps/details?id=com.ledger.live',
-    ios: 'https://apps.apple.com/us/app/ledger-live-web3-wallet/id1361671700',
-    mobile: 'https://www.ledger.com/ledger-live',
-    qrCode: 'https://ledger.com/ledger-live',
-  },
-  createConnector: () => {
-    const connector = getWalletConnectConnector({ projectId, chains });
-
-    return {
-      connector,
-      mobile: {
-        getUri: async () => {
-          const { uri } = (await connector.getProvider()).connector;
-          return isAndroid()
-            ? uri
-            : `ledgerlive://wc?uri=${encodeURIComponent(uri)}`;
+  ...options
+}: LedgerWalletOptions & InjectedConnectorOptions): Wallet => {
+  const initialChainId = chains[0].id;
```

Comment:
Is there a better way to get the initial chain the dApp is trying to connect to?